test/unit/store/cart-store.js

```diff
@@ -0,0 +1,31 @@
+import { Dispatcher } from 'consus-flux';
+import CartStore from '../../../.dist/store/cart-store';
+import { assert } from 'chai';
+
+describe('CartStore', () => {
+
+    it('should instantiate without any items', () => {
+        assert.strictEqual(CartStore.getItems().length, 0);
+    });
+
+    it('should add item to list', () => {
+        Dispatcher.handleAction('CHECKOUT_ITEM_FOUND',{
```

Comment:
Maybe this should be called 'CHECKOUT_ITEM_ADDED', since we're not really "searching" for an item here. Tests look good, though.